lib/po.js

```diff
@@ -234,7 +236,24 @@ PO.parse = function (data) {
     return po;
 };
 
-PO.Item = function () {
+PO.parsePluralForms = function (pluralFormsString) {
+    var results = (pluralFormsString || '')
+        .split(';')
+        .reduce(function (acc, keyValueString) {
+            var trimmedString = keyValueString.trim();
+            var equalsIndex = trimmedString.indexOf('=');
+            var key = trimmedString.substring(0, equalsIndex).trim();
+            var value = trimmedString.substring(equalsIndex + 1).trim();
+            acc[key] = value;
+            return acc;
+        }, {});
+    return {
+        nplurals: results.nplurals,
+        plural: results.plural
+    };
+};
+
+PO.Item = function (nplurals) {
```

Comment:
After leaving this for a while and thinking about it, again, IMHO it is fine to go with this API. So no, I don't have a strong preference, here. For readability and extensibility, I'd change it to an object instead of int. So the calling code explicitly sets the desired value:

```
let item = new PO.Item({ nplurals: 6 });
```

But on the other hand this would look strange in the current situation: `ìtem = new PO.Item({ nplurals: nplurals });`.

So I'd say, leave it as is for now and we change it, if needed. Since it's not directly covered by any tests, we can consider it *private* API.